src/Features/LanguageServer/Protocol/Handler/SemanticTokens/TokenModifiers.cs

```diff
@@ -16,5 +16,6 @@ internal enum TokenModifiers
         None = 0,
         Static = 1,
         ReassignedVariable = 2,
+        ObsoleteSymbol = 4,
```

Comment:
This doesn't appear to be a known LSP modifier - https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#semanticTokenModifiers

This probably needs to use the existing token modifier (`deprecated`), or we have to add custom client side support for the modifier
See https://code.visualstudio.com/api/language-extensions/semantic-highlight-guide#custom-token-types-and-modifiers

Reply:
➡️ Now updated to use the `deprecated` modifier